packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/models/models.ts

```diff
@@ -825,66 +690,197 @@ export interface _OperationListResult {
+  /** Represents an accepted operation. */
+  Accepted = '"Accepted"',
+}
+
+export type ProvisioningState =
+  | "Succeeded"
+  | "Failed"
+  | "Canceled"
+  | "Provisioning"
+  | "Updating"
+  | "Deleting"
+  | "Accepted";
+
+/** The data type state */
+export enum ControlStateKnownValues {
+  /** Field to enable a setting. */
+  Enabled = '"Enabled"',
+  /** Field to disable a setting. */
+  Disabled = '"Disabled"',
+}
+
+export type ControlState = "Enabled" | "Disabled";
+
+/** Specifies the default action of allow or deny when no other rules match. */
+export enum DefaultActionKnownValues {
+  /** Represents allow action. */
+  Allow = '"Allow"',
+  /** Represents deny action. */
+  Deny = '"Deny"',
+}
+
+export type DefaultAction = "Allow" | "Deny";
+
+/** Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed). */
+export enum ManagedServiceIdentityTypeKnownValues {
+  /** No managed identity. */
+  None = '"None"',
+  /** System assigned managed identity. */
+  SystemAssigned = '"SystemAssigned"',
+  /** User assigned managed identity. */
+  UserAssigned = '"UserAssigned"',
+  /** System and user assigned managed identity. */
+  SystemAndUserAssigned = '"SystemAssigned, UserAssigned"',
+}
+
+export type ManagedServiceIdentityType =
+  | "None"
+  | "SystemAssigned"
+  | "UserAssigned"
+  | "SystemAssigned, UserAssigned";
+
+/** The kind of entity that created the resource. */
+export enum CreatedByTypeKnownValues {
+  /** The entity was created by a user. */
+  User = '"User"',
+  /** The entity was created by an application. */
+  Application = '"Application"',
+  /** The entity was created by a managed identity. */
+  ManagedIdentity = '"ManagedIdentity"',
+  /** The entity was created by a key. */
+  Key = '"Key"',
+}
+
+export type CreatedByType = "User" | "Application" | "ManagedIdentity" | "Key";
+
+/** The data type state */
+export enum DataProductUserRoleKnownValues {
+  /** Field to specify user of type Reader. */
+  Reader = '"Reader"',
+  /**
+   * Field to specify user of type SensitiveReader.
+   * This user has privileged access to read sensitive data of a data product.
+   */
+  SensitiveReader = '"SensitiveReader"',
+}
+
+export type DataProductUserRole = "Reader" | "SensitiveReader";
+
+/** The data type state */
+export enum DataTypeStateKnownValues {
+  /** Field to specify stopped state. */
+  Stopped = '"Stopped"',
+  /** Field to specify running state. */
+  Running = '"Running"',
+}
+
+export type DataTypeState = "Stopped" | "Running";
+
+/** The intended executor of the operation; as in Resource Based Access Control (RBAC) and audit logs UX. Default value is "user,system" */
+export enum OriginKnownValues {
+  /** Indicates the operation is initiated by a user. */
+  user = '"user"',
+  /** Indicates the operation is initiated by a system. */
+  system = '"system"',
+  /** Indicates the operation is initiated by a user or system. */
+  "user,system" = '"user,system"',
```

Comment:
we should check if this is expected as we have decided to use typespec enum name as ts enum name.